runtime/queries/xit/highlights.scm

```diff
@@ -0,0 +1,11 @@
+(headline) @markup.heading
+
+(open_checkbox) @markup.list
+(ongoing_checkbox) @keyword.control
+(checked_checkbox) @diff.plus
+(obsolete_checkbox) @hint
+
+(checked_task) @hint
+(obsolete_task) @hint
```

Comment:
I'm not very familiar with xit but `@comment.unused` might be a good scope for these. That scope is usually used for languages with pattern matching where a variable is underscored when it's unused:

```elixir
{_x, y} = {100, 50}
y * 2
```

What do you think?

Reply:
yes, you are right, I have amended the changes